internal/settings/globalSettings/defaults.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package globalSettings
```

Comment:
I would rename the package. Repeating parent package name in Go is called stuttering https://go.dev/doc/effective_go#package-names
I suggest just having `internal/settings/defaults.go`

Reply:
Don't need globalSettings package anymore actually now that I'm using viper instead of my previous implementation. Removed.